src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -75,11 +78,16 @@ int main(int argc, const char *argv[]) {
                                           const std::string &argument) {
              numReservedFunctionPointers = std::stoi(argument);
            })
-      .add("--global-base", "", "Where lld started to place globals",
+      .add("--global-base", "", "Where lld should start to place static globals",
```

Comment:
This command seems wrong.  lld has already assigned all its globals before finalize is called.

Reply:
I see we pass the global base to both wasm-emscripten-finalize and to lld - is lld doing it in both bitcode linking and wasm object files? If so I can remove the flag to finalize.

Reply:
The only reason I can see that finalize needs to know this is because it wants to write "staticBump" to the metadata output about the binary.   as long as we need staticBump we will to continue to pass this.